pptxProject.py

```diff
@@ -0,0 +1,141 @@
+import asyncio
+import os
+import time
+
+import aiohttp
+from flask import json
+from pptx import Presentation
+import openai
+
+
+REQ = "Please provide an English summary and brief explanation of the text on this slide, " \
+      "along with a recommended website for further learning. " \
+      "If the slide does not contain any text, please do not provide an answer. " \
+      "Please format your response as an educational paragraph.\n"
+KEY = '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'
+
+messages = []
+messages.append({"role": "system", "content": f"you are a helpful AI that is explaining a PowerPoint presentation."})
+openai.api_key = KEY
```

Comment:
Use constants with uppercase, it may be better to use the messages inside a function